extensions/resource-deployment/notebooks/azurevm/create-sqlvm.ipynb

```diff
@@ -0,0 +1,396 @@
+{
```

Comment:
the `SQL VM` deployment notebook is also part of this PR?

Reply:
This is from Aasim's branch, it shouldn't be reviewed